From 596c0e73f0490cd5d53c33ee8e7717ab7716ddef Mon Sep 17 00:00:00 2001
From: Fenghua Yu <fenghua.yu@intel.com>
Date: Mon, 19 Apr 2021 21:49:57 +0000
Subject: [PATCH 0221/1646] [Backport] Documentation/admin-guide: Add bus lock
 ratelimit

mainline inclusion
from mainline-v5.14-rc1
commit 9d839c280b64817345c2fa462c0027a9bd742361
category: feature
bugzilla: https://gitee.com/openeuler/intel-kernel/issues/I5G10C

Reference: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=9d839c280b64817345c2fa462c0027a9bd742361

--------------------------------

Since bus lock rate limit changes the split_lock_detect parameter,
update the documentation for the change.

Signed-off-by: Fenghua Yu <fenghua.yu@intel.com>
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Reviewed-by: Tony Luck <tony.luck@intel.com>
Link: https://lore.kernel.org/r/20210419214958.4035512-4-fenghua.yu@intel.com

(cherry picked from commit 9d839c280b64817345c2fa462c0027a9bd742361)
Signed-off-by: Ethan Zhao <haifeng.zhao@linux.intel.com>
Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>
---
 Documentation/admin-guide/kernel-parameters.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Documentation/admin-guide/kernel-parameters.txt b/Documentation/admin-guide/kernel-parameters.txt
index 39303efa10e9..dccefaa14fd0 100644
--- a/Documentation/admin-guide/kernel-parameters.txt
+++ b/Documentation/admin-guide/kernel-parameters.txt
@@ -5397,6 +5397,14 @@
 				  exception. Default behavior is by #AC if
 				  both features are enabled in hardware.
 
+			ratelimit:N -
+				  Set system wide rate limit to N bus locks
+				  per second for bus lock detection.
+				  0 < N <= 1000.
+
+				  N/A for split lock detection.
+
+
 			If an #AC exception is hit in the kernel or in
 			firmware (i.e. not while executing in user mode)
 			the kernel will oops in either "warn" or "fatal"
-- 
2.17.1

